.github/workflows/tests.yml

```diff
@@ -77,12 +77,10 @@ jobs:
                   .github/check-executable-files.sh ./
 
             - name: Check commit message
-              env:
-                  BRANCH: ${{ github.event.pull_request.base.sha }}
-                  PR_SHA: ${{ github.event.pull_request.head.sha }}
-              if: github.event.pull_request.head.sha != '' && github.event.pull_request.base.sha != ''
               run: |
-                  git-tools/commit-msg-hook-range.sh $BRANCH..$PR_SHA
+                  # Get previous merge commit to get commit range from
+                  export BRANCH=$(git rev-list --merges -n 1 HEAD^1)
+                  git-tools/commit-msg-hook-range.sh $BRANCH..$GITHUB_SHA
```

Comment:
Where does `$GITHUB_SHA` comes from?

Reply:
It's default GA env var https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables